src/main/kotlin/me/gserv/fabrikommander/Common.kt

```diff
@@ -27,7 +27,18 @@ object Common : ModInitializer {
         HomesCommand(dispatcher).register()
         SetHomeCommand(dispatcher).register()
 
-        // Misc comands
+        // TPA commands
```

Comment:
Don't do this. Instead, you can open a PR to my fork.

Reply:
Yeah, I forgot that the file had these lines, I removed them when I realized and only included the RtpCommand line

Reply:
You can mark this as resolved - please do that